README.md

```diff
@@ -39,8 +39,11 @@ npm run action src/www/storybook
 Additional requirements for Android:
 
 - [Android Studio 2020.3.1+](https://developer.android.com/studio)
+  - Optional for building, but useful for development
 - [Latest Android Sdk Commandline Tools](https://developer.android.com/studio/command-line)
-- Android SDK 30 (with build-tools) via commandline `sdkmanager "platforms;android-30" "build-tools;30.0.3"`
+  - Place it at `~/Android/Sdk/cmdline-tools/latest`
+- Android SDK 30 (with build-tools) via commandline `~/Android/Sdk/cmdline-tools/latest/bin/sdkmanager "platforms;android-30" "build-tools;30.0.3"`
+  - Set up the environment: `export ANDROID_SDK_ROOT=~/Android/Sdk`
```

Comment:
We can't assume this is where they've installed their SDK - mine certainly isn't there

Reply:
I'm not really assuming that. It's just a recommendation. We have no recommendation otherwise and it was a pain to get it set up.
It's aligned with the recommendation I added to outline-go-tun2socks: https://github.com/Jigsaw-Code/outline-go-tun2socks#set-up